tools/chef/cookbooks/px_dev/recipes/packaging.rb

```diff
@@ -0,0 +1,27 @@
+# Copyright 2018- The Pixie Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Packaging utilities only need to run on Linux
+if node.platform_family?('debian')
+  apt_package ['ruby-dev', 'rpm'] do
+    action :upgrade
+  end
+
+  execute 'install_fpm' do
+    command '/usr/bin/gem install fpm'
+    action :run
```

Comment:
I tried to use the `gem_package` resource to avoid running a shell command. Unfortunately this was installing the gem within the chef ruby (confirmed with `opensnoop` as the build ran). Even if `gem_binary` specified this gem binary (`/usr/bin/gem`), I couldn't find a way to get it to work.